app/code/Magento/InventoryApi/Test/Api/SourceRepository/CarrierLinkManagementTest.php

```diff
@@ -0,0 +1,202 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryApi\Test\Api\SourceRepository;
+
+use Magento\Framework\Webapi\Rest\Request;
+use Magento\InventoryApi\Api\Data\SourceCarrierLinkInterface;
+use Magento\InventoryApi\Api\Data\SourceInterface;
+use Magento\TestFramework\TestCase\WebapiAbstract;
+
+class CarrierLinkManagementTest extends WebapiAbstract
+{
+    /**#@+
+     * Service constants
+     */
+    const SOURCE_PATH = '/V1/inventory/source';
+    const SOURCES_PATH = '/V1/inventory/sources';
```

Comment:
why do we have two paths here? for the single and plural form
it should be just one - `/V1/inventory/sources`

the same, for example, we have for a product
```xml
    <route url="/V1/products" method="GET">
        <service class="Magento\Catalog\Api\ProductRepositoryInterface" method="getList"/>
        <resources>
            <resource ref="Magento_Catalog::products" />
        </resources>
    </route>
    <route url="/V1/products/:sku" method="GET">
        <service class="Magento\Catalog\Api\ProductRepositoryInterface" method="get"/>
        <resources>
            <resource ref="Magento_Catalog::products" />
        </resources>
    </route>
```
and other entities